src/core/obj.js

```diff
@@ -877,7 +879,7 @@ class Catalog {
     const obj = this._catDict.get("Names");
 
     let javaScript = null;
-    function appendIfJavaScriptDict(jsDict) {
+    function appendIfJavaScriptDict(name, jsDict) {
```

Comment:
Unless I'm mistaken, the changes here will affect the existing format used for the `getJavaScript` method in the API; which constitutes a *breaking* API-change and that's very much not allowed!

The fact that you had to change an existing unit-test is definitely a red-flag, and you've most likely also broken the `_initializeAutoPrint` method in the viewer with these changes.